openapi.yaml

```diff
@@ -26,7 +28,552 @@ tags:
+    parameters:
+      - name: projectId
+        in: path
+        required: true
+        schema:
+          $ref: '#/components/schemas/ProjectId'
+    get:
+      tags:
+        - admins
+        - Project
+      summary: Get Project by projectId
+      operationId: GetProjectById
+      description: |
+        List Index by projectId
+      responses:
+        '200':
+          description: Array of registered projects
+    put:
+      tags:
+        - admins
+        - Project
+      summary: Update a Project
+      operationId: UpdateProject
+      requestBody:
+        description: Project information
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/UpdateProjectRequest'
+      description: |
+        Update a Project
+      responses:
+        '200':
+          description: Project updated
+    delete:
+      tags:
+        - admins
+        - Project
+      summary: Delete a Project
+      operationId: DeleteProject
+      description: |
+        Delete a Project
+      responses:
+        '200':
+          description: Project deleted
+  /api/indexes:
+    get:
+      tags:
+        - admins
+        - Index
+      summary: List all indexes
+      operationId: ListIndex
+      description: |
+        List all indexes
+      responses:
+        '200':
+          description: Array of registered indexes
+    post:
+      tags:
+        - admins
+        - Index
+      summary: Create a new Index
+      operationId: CreateIndex
+      description: |
+        Create new Index
+      requestBody:
+        description: Index information
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CreateIndexRequest'
+      responses:
+        '200':
+          description: Index added
+  /api/indexes/{indexId}:
+    parameters:
+      - name: indexId
+        in: path
+        required: true
+        schema:
+          $ref: '#/components/schemas/IndexId'
+    get:
+      tags:
+        - admins
+        - Index
+      summary: Get Index by indexId
+      operationId: GetIndexById
+      description: |
+        List Index by indexId
+      responses:
+        '200':
+          description: Array of registered indexes
+    put:
+      tags:
+        - admins
+        - Index
+      summary: Update an Index
+      operationId: UpdateIndex
+      requestBody:
+        description: Index information
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/UpdateIndexRequest'
+      description: |
+        Update an Index
+      responses:
+        '200':
+          description: Index updated
+    delete:
+      tags:
+        - admins
+        - Index
+      summary: Delete an Index
+      operationId: DeleteIndex
+      description: |
+        Delete an Index
+      responses:
+        '200':
+          description: Index deleted
+  /api/compute/platforms:
+    get:
+      tags:
```

Comment:
Create a built-in tag for build-in workspace APIs?

Reply:
Actually, I am leaning towards removing this api from docs if it is built-in related. What do you think?

Reply:
Yes, we don't want to build more assets for built-in, just adds more work when we deprecate them. I was speculating that it's build-in workspace from the path, please do double check if you're going to remove them.